README.md

```diff
@@ -16,7 +16,7 @@ This bundle is a semantic configuration and service provider for the AWS PHP SDK
 ``` php
 $bundles = array(
 	// ...
-	new Uecode\Bundle\AmazonBundle\UecodeAmazonBundle()
+	new Uecode\Bundle\AmazonBundle\UecodeAmazonBundle(),
```

Comment:
disagree though, i think it was more practical to just copy paste and avoids diffs